src/mixs/schema/mixs.yaml

```diff
@@ -3813,12 +3813,12 @@ slots:
     required: true
   abs_air_humidity:
     annotations:
-      Preferred_unit: gram per gram, kilogram per kilogram, kilogram, pound
-    description: Actual mass of water vapor - mh20 - present in the air water vapor
-      mixture
+      Preferred_unit: gram per gram, kilogram per kilogram, kilogram, pound, gram per cubic meter, kilogram per cubic meter
+    description: Actual mass of water vapor present in the air water vapor mixture. Can be calculated via mass of water vapor divided by the volume of the air and water vapor mixture.
```

Comment:
@turbomam  is there an attribute better suited for the "Can be calculated via mass of water vapor divided by the volume of the air and water vapor mixture."

part of this ? Since it's not really description. Comments?

Reply:
yes, definitely comments